src/framed/std/core.cljc

```diff
@@ -130,3 +139,15 @@
        (loop []
          ~@body
          (recur)))))
+
+(defn try-to-edn
```

Comment:
(Swooped the PR out from under you , now https://github.com/framed-data/std/pull/13)

They could be in `serialization`, but I don't know if that's appropriate - they're really variations of `pr-str` and `"un-pr-str"`, and I considered naming them as such